testing/MLDB-180-basic-join.js

```diff
@@ -258,7 +258,7 @@ for (var i in funcs) {
 // MLDB-1088
 
 expected = [
-   [ "_rowName", "test1.x", "test1.y", "test1.z" ],
+   [ "_rowName", "x", "y", "z" ],
```

Comment:
No, it's not generalized (we should work on doing that, as it's currently all over the place).  The change I made caused an existing codepath to be used for the two cases, and so it had to change either one way or the other.  Changing it this way led to the smallest number of tests being changed.